spec/20-http_request_header_format.md

```diff
@@ -297,7 +297,7 @@ The `tracestate` value is the concatenation of trace graph key/value pairs
 
 Example: `vendorname1=opaqueValue1,vendorname2=opaqueValue2`
 
-Only one entry per key is allowed because the entry represents that last position in the trace. Hence vendors must overwrite their entry upon reentry to their tracing system.
```

Comment:
"because the entry represents that last position in the trace." had been called out [here](https://github.com/w3c/trace-context/issues/446#issuecomment-759073256) by @yurishkuro as not making a lot of sense, and I agree. Hence, I removed that part.

Also, I think update could be a better phrase instead of overwrite because we also use that in the section "Mutating the traceparent Field". 

Finally "upon reentry to their tracing system" is really unwieldy, IMO.